pennylane/ops/qubit/state_preparation.py

```diff
@@ -169,20 +169,20 @@ class StatePrep(StatePrepBase):
     ndim_params = (1,)
     """int: Number of dimensions per trainable parameter of the operator."""
 
-    def __init__(self, state, wires, id=None):
+    def __init__(self, state, wires, id=None, skip_norm_validation=False):
```

Comment:
```suggestion
    def __init__(self, state, wires, id=None, validate_norm=True):
```
I'm fine either way, but "skip" is essentially a layer of negation, so this removes a layer of negation we have to keep in our brain.

Reply:
I agree, I'll change it to `validate_norm`.